atlas/templates/cmd/server/main.go.gotmpl

```diff
@@ -110,6 +111,8 @@ func ServeExternal(logger *logrus.Logger) error {
 		server.WithGateway(
 			gateway.WithGatewayOptions(
 				runtime.WithForwardResponseOption(forwardResponseOption),
+				runtime.WithIncomingHeaderMatcher(whitelistStringsHeaderMatcher(
```

Comment:
Correct me if i wrong but as i understand, only headers passed in function accepted, but as i saw [there](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/mux.go#L63), grpc-gateway has default header matching rules, From my point of view we shouldn't replace logic of Header matching just extend.